GitHub tutorial/github_tutorial.md

```diff
@@ -70,6 +70,24 @@ Wanneer je op je eigen laptop/pc werkt aan bestanden, is "origin" de alias voor
 **upstream**
 Wanneer je op je eigen laptop/pc werkt aan bestanden, is "upstream" de alias voor de GitHub (remote) repository van VNG-Realisatie. Dit gebruik je wanneer je de lokale kopie van de repository wilt actualiseren met alle door anderen doorgevoerde wijzigingen (via een `git pull upstream master`).
 
+
+**lokale files excluden van commits** Hiervoor kan niet de .gitignore worden gebruikt omdat dit bestand ook wordt ingecheckt en dit zou er dan voor zorgen dat de GitHub Actions workflows de gegenereerde bestanden niet kan committen.
+In een git repo is er een .git/info/exclude file die hetzelfde werkt als de .gitignore. Alleen wordt deze niet gecommit waardoor het alleen voor de lokale repo geldt. De volgende regels moeten worden toegevoegd om de gegenereerde bestanden te excluden voor commit.
+
+- specificatie/genereervariant/**
+- code/**
+- test/**
+- openapitools.json
+
+Omdat de gegenereerde bestanden al in git worden getracked, moet een extra actie worden uitgevoerd om de gegenereerde bestanden lokaal niet meer te tracken. Dit moet met de volgende bash statements:
+
+- git ls-files -z code | xargs -0 git update-index --assume-unchanged
+- git ls-files -z specificatie/genereervariant | xargs -0 git update-index --assume-unchanged
+
+Om een specifieke file te untracken moet de volgende statement worden gebruikt: git update-index --assume-unchanged <filepath> bijv. git update-index --assume-unchanged test/BRP-Bevragen-postman-collection.json.
+Hiermee worden merge conflicts in de gegenereerde bestanden voorkomen.
```

Comment:
Ik neem aan dat deze laatste zin niet alleen slaat op deze laatste alinea maar op de gehele paragraaf. Zo ja, voeg dan een witregel in.

Reply:
Aangepast